src/FramedClient.ts

```diff
@@ -109,13 +110,14 @@ class FramedClient {
         })
     }
 
-    public async uploadDocument(documentId: string, file: File | AMSFileInfo, chatToken: OmnichannelChatToken | null = null): Promise<void> {
+    public async uploadDocument(documentId: string, file: File | AMSFileInfo, chatToken: OmnichannelChatToken | null = null, supportedImagesMimeTypes?: string[]): Promise<void> {

```

Comment:
Please add `[]` as default value for `supportedImagesMimeTypes` for backward compatibility.

Reply:
done